upstream/roles/build_operator_index_sha/tasks/main.yml

```diff
@@ -66,18 +66,18 @@
       set_fact:
         result_rc: true
       when: ((sis_index_rm_rc.rc is defined and sis_index_rm_rc.rc == 0) and rc_last|bool)
-    - name: "Prints message that bundles were added to catalog"
+    - name: "Prints message that bundles were removed from catalog"
       debug:
         msg: "Operator '{{ sis_package_name }}' were removed and push to catalog '{{ sis_index_image_output }}' : [OK]"
       when:
         - result_rc|bool
-    - name: "Print failing msg that bundle was not added to catalog or not pushed"
+    - name: "Print failing msg that bundle was not removed to catalog or not pushed"
       debug:
         msg: "Operator '{{ sis_package_name }}' were removed and push to catalog '{{ sis_index_image_output }}' : [FAIL]"
       when:
         - not result_rc|bool
         - (strict_mode is undefined or not strict_mode|bool)
-    - name: "Failing when bundle was not added to catalog or not pushed"
```

Comment:
Could you add empty line between tasks (before `-name`)?
It should be simpler to understand the code.
Cause it will be to simpler find out which condition belongs to which task
And do that also on other places.

Reply:
@mvalarh bump